dialogue-core/src/main/java/com/palantir/dialogue/core/StickyEndpointChannels2.java

```diff
@@ -172,7 +202,12 @@ public String toString() {
         private final class InFlightCallSuccessTransformer implements Function<Response, Response> {
             @Override
             public Response apply(Response response) {
-                successfulCall(response);
+                try {
+                    successfulCall(response);
+                } catch (Throwable t) {
```

Comment:
Is this overly paranoid?

Reply:
There's no great way for us to know, unfortunately